### ISSUE TITLE
TTNN Op Logging

### ISSUE THREAD

**Issue by mywoodstock** [tenstorrent-metal/tt-metal#5866]
Its useful to have structured logs output from model runs in TTNN, may be conditionally on an env var. Currently we have Debug logs printed from C++ host code during the run, which is too verbose and hard to parse.

**Comment by davorchap** [comment]
Yes please , one liner log per ttnn OP.

**Comment by zzigler-tt** [comment]
@fahmedchanTT Can you please review and determine if this is still a to-do item, if so please reassign to a team member. 
If not, please close this ticket. Thanks

**Comment by fahmedchanTT** [comment]
This is solved with the Tracy profiler and tt-nn visualizer. Closing.
